__tests__/test.js

```diff
@@ -40,8 +39,35 @@ describe('/api/topics', () => {
 
 describe('/api', () => {
     test('GET request should respond with 200 and an object containing all endpoints with a description', () => {
-        return request(app).get('/api').then(({body}) => {
-            expect(body.endpoints).toEqual(endpoints)
-            })
+    return request(app).get('/api').expect(200).then(({body}) => {
+        expect(body.endpoints).toEqual(endpoints)
+        })
+    })
+})
+
+describe('/api/articles/:article_id', () => {
+    test('GET request should respond with 200 and an object', () => {
+        return request(app).get('/api/articles/4').expect(200).then(({body}) => {
+            const {article} = body
+            expect(typeof article).toBe('object')
         })
     })
+    test(`GET request should respond with an object containing the folowing properties:author, title, article_id, body, topic, created_at, votes, article_img_url`, () => {
+        return request(app).get('/api/articles/4').then(({body}) => {
+            const {article} = body
+            expect(article).toHaveProperty('author')
+            expect(article).toHaveProperty('title')
+            expect(article).toHaveProperty('body')
+            expect(article).toHaveProperty('topic')
+            expect(article).toHaveProperty('created_at')
+            expect(article).toHaveProperty('votes')
+            expect(article).toHaveProperty('article_img_url')
+            expect(article.article_id).toBe(4)
+        })
+    })
+    test('GET request should respond with 404 and a msg of invalid article_id if invalid article_id is passed', () => {
+        return request(app).get('/api/articles/10000086').expect(404).then(({body}) => {
+            expect(body.msg).toBe('invalid article ID')
+        })
+    })
```

Comment:
This is a great test for if the number cannot be found for `article_id`

What about another scenario where the `article_id` is invalid ( incorrect data type "/api/articles/banana" ) ?